packages/__examples__/.storybook/stories/TooltipPositioning.tsx

```diff
@@ -0,0 +1,213 @@
+import { ApplyLocale } from '@instructure/ui-i18n'
+import React from 'react'
+import { Alert } from '@instructure/ui-alerts/src/Alert'
```

Comment:
I think (almost) all these components could be imported from the `ui` package. Also, we avoid importing from `/src` everywhere else too.

Reply:
Yep, I missed this. Also I think the ESLint doesn't process these files for some reason